src/components/Inputs/TextArea/TextArea.tsx

```diff
@@ -217,6 +237,27 @@ export const TextArea: FC<TextAreaProps> = React.forwardRef(
       debouncedChange(e);
     };
 
+    const getIconSize = (): IconSize => {
+      let iconSize: IconSize;
+      if (largeScreenActive) {
```

Comment:
would be nice to have a hook which simply returns current screen size ('lg', 'md', 'sm', 'xs' etc), and then a mapping could be used instead of if/else. I would guess this would be useful in other components?

Reply:
We have such a hook, and this could be a mapping as the properties here are returned by a hook, but will consider this for a future change as it's code style only.